_inc/client/components/jumpstart/index.jsx

```diff
@@ -43,7 +43,7 @@ class JumpStart extends Component {
 				onClick={ this.props.jumpStartSkip }
 				className="jp-jumpstart__skip-link"
 			>
-				{ __( 'Skip and explore features' ) }
+				{ __( 'Skip, and explore features individually' ) }.
```

Comment:
Genuinely curious about this: is it good practice to leave non-important punctuation out of the translatable string?

Reply:
You got a point. I'll put it inside.